multiqc/modules/damageprofiler/damageprofiler.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from DamageProfiler """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import json
+import pprint
+
+from multiqc import config
+from multiqc.plots import  linegraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+        """
+        damageprofiler module class
+        """
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='DamageProfiler:',
+        anchor='damageprofiler',
+        href='https://github.com/Integrative-Transcriptomics/DamageProfiler',
+        info="a tool to determine damage patterns on ancient DNA.")
+
+        # Init empty dictionaries
+
+        self.threepGtoAfreq_data = dict()
+        self.fivepCtoTfreq_data = dict()
+        self.lgdist_fw_data = dict()
+        self.lgdist_rv_data = dict()
+
+        # Find and load JSON file
+        for f in self.find_log_files('damageprofiler',filehandles=True):
+            self.parseJSON(f)
+
+        # Filter to strip out ignored sample names
+        self.threepGtoAfreq_data         =   self.ignore_samples(self.threepGtoAfreq_data)
+        self.fivepCtoTfreq_data          =   self.ignore_samples(self.fivepCtoTfreq_data)
+        self.lgdist_fw_data   =   self.ignore_samples(self.lgdist_fw_data)
+        self.lgdist_rv_data      =   self.ignore_samples(self.lgdist_rv_data)
+
+
+        # Write parsed data to a file
+        #self.write_data_file(self.threepGtoAfreq_data, 'multiqc_damageprofiler_3pGtoAfreq')
```

Comment:
I usually don't write plot data to files. There is already an option to do this from core MultiQC.

Reply:
Removed this :-)